pkg/proxy/config/config.go

```diff
@@ -31,28 +31,21 @@ type Operation int
 
 // These are the available operation types.
 const (
-	SET Operation = iota
-	ADD
+	ADD Operation = iota
```

Comment:
We use add and remove consistently in event informers, I would prefer not to drift into othe syntax.  This is ultimately referring to membership change in a set.